Program.cs

```diff
@@ -15,252 +15,273 @@ public static void printsContentsRefrigerator(List<Refrigerator> allfrige)
+            Console.WriteLine("1. " + (Item.TypeItem)0);
+            Console.WriteLine("2. " + (Item.TypeItem)1);
+            userInput = Console.ReadLine();
+            if (!(int.TryParse(userInput, out typeItem)) || typeItem < 1 || typeItem > 2)
+                return -1;
+            return --typeItem;
         }
-        public static String checkNameItemInput()
+        public static String GetNameItemInput()
         {
             string nameInput = "";
             Console.WriteLine("Name of the Item");
             nameInput = Console.ReadLine();
             while (nameInput.Length <= 0 || nameInput.Length > 50)
-                {
-                    Console.WriteLine("Please,enter valid input");
-                    nameInput = Console.ReadLine();
-                }
+            {
+                Console.WriteLine("Please,enter valid input");
+                nameInput = Console.ReadLine();
+            }
             return nameInput;
 
-            
+
         }
-        public static DateTime checkExpiryDatedInput()
+        public static object GetExpiryDatedInput()
         {
             string expiryDateInput;
             DateTime expiryDate = DateTime.Now;
             Console.WriteLine("Please enter the expiry date of the item in this format (MM/dd/yyyy):");
             Console.WriteLine("Dont enter a date before the year 2000");
             expiryDateInput = Console.ReadLine();
-            while ((!(DateTime.TryParseExact(expiryDateInput, "MM/dd/yyyy", CultureInfo.InvariantCulture, DateTimeStyles.None, out expiryDate))) || expiryDate.Year < 2000)
+            if ((!(DateTime.TryParseExact(expiryDateInput, "MM/dd/yyyy", CultureInfo.InvariantCulture, DateTimeStyles.None, out expiryDate))) || expiryDate.Year < 2000)
             {
-                Console.WriteLine("Please,enter valid input ");
-                expiryDateInput = Console.ReadLine();
+                return -1;
 
             }
             return expiryDate;
         }
-        public static int checkSizeItemInput()
+        public static int GetSizeItemInput()
         {
             string sizeItemInput;
             int sizeItem;
             Console.WriteLine("Please enter the size of thie item");
             Console.WriteLine("Dont insert an item larger than 1000 ");
             sizeItemInput = Console.ReadLine();
-            while ((!int.TryParse(sizeItemInput, out sizeItem)) || sizeItem <= 0 || sizeItem > 1000)
+            if ((!int.TryParse(sizeItemInput, out sizeItem)) || sizeItem <= 0 || sizeItem > 1000)
             {
-                Console.WriteLine("Please,enter valid input ");
-                sizeItemInput = Console.ReadLine();
+                return -1;
 
             }
             return sizeItem;
-        }           
-        static void Main(string[] args)
+        }
+        public static void CreatDate(List<Item> allItems, List<Shelf> allIShelfes, List<Refrigerator> allIFriger)
```

Comment:
change CreatDate name to more indicative name, and inside the functions, use loops to create the item with random dates, its more efficient and less code.